server/pairing/client.go

```diff
@@ -69,6 +76,12 @@ func NewBaseClient(c *ConnectionParams) (*BaseClient, error) {
 
 	if serverCert == nil {
 		certErrs = fmt.Errorf("failed to connect to any of given addresses. %w", certErrs)
+		if retryCount < maxRetries {
```

Comment:
This makes it total of 4 attempts, is this right?

Reply:
yes if it all failed, do you think it's too many? 🤔 @igor-sirotin

Reply:
It's fine by me. I just got it wrong at first from "retries := 3" 🙂